services/web/client/source/class/osparc/component/form/Auto.js

```diff
@@ -448,12 +452,16 @@ qx.Class.define("osparc.component.form.Auto", {
         Object.assign(s, s.contentSchema);
       }
 
-      if (s.defaultValue) {
-        if (!s.set) {
-          s.set = {};
-        }
+      if (!s.set) {
+        s.set = {};
+      }
+      if ("defaultValue" in s) {
         s.set.value = s.defaultValue;
       }
+      // Content Schema
+      if ("default" in s) {
```

Comment:
so default in content schema overrides ``defaultValue`` 👍

Reply:
> Great! Why didn't you add the example in the. `demo_units.py` as well?

That's where I added them, but I didn't commit the changes. May I?